cmd/git/main.go

```diff
@@ -129,6 +131,19 @@ func runGitClone(ctx context.Context) error {
 		}
 	}
 
+	if flagValues.resultFileCommitAuthor != "" {
+		output, err := git(ctx, "-C", flagValues.target, "log", "-1", "--pretty=format:%an")
+		if err != nil {
+			return err
+		}
+
+		if err = ioutil.WriteFile(
+			flagValues.resultFileCommitAuthor, []byte(output), 0644,
+		); err != nil {
```

Comment:
Open to debate, just wanted to see the suggestion diff to get a feeling that it would still be readable even in compact style.
```suggestion
		if err = ioutil.WriteFile(flagValues.resultFileCommitAuthor, []byte(output), 0644); err != nil {
```